### ISSUE TITLE
L2 file

### ISSUE THREAD

**Issue by melvanhove** [martin-obs/OVERLAP_PROBE_EPROFILE#1]
In the script run_process_L1.py, if I comment the line ```L2_file = "L2." ``` it doesn't seem to change the output, and I haven't found where the variable L2_file is used in the scripts.
I used the command ```pyhton run_process.py```, but I'm probably missing something...

UPDATE : I was looking for a netcdf but I've just seen the file ```ov_results_TUB200032_2022-07-02.csv```, which I can work with, sorry for bothering!..

**Comment by martin-obs** [comment]
Ah! the mention of L2 is a left over from our original idea to see if this code would work with L2 data. You are quite right that it is not used in the code anywhere. I will delete it. 

Great news that the code runs and produces a result. Fingers crossed it is close to the correct.

**Comment by melvanhove** [comment]
I see, thanks! 
So, I could compare the outputs:
The precision of the ```.csv``` file is ```10e-19``` and the precision of the ```.mat``` file is ```10e-13```, so I rounded the values of the ```.csv``` file to ```10e-13```.
Then I calculated the relative difference between the rounded outputs, and none of the differences is higher than ```10e-6```, so with a precision of ```10e-6``` the output are identical. I'm pretty sure it's enough, even though I will have to compare the impact of the models to be sure. 
I'm making a general code to compare a larger amount of data; I will run it on all L1 available for Qualair.
Thanks again for your work !
